tests/unit/test_webhook.py

```diff
@@ -83,6 +83,11 @@ def setUp(self):
         **Tested Another Browser**: Yes
         """  # noqa
 
+        self.issue_body6 = u"""
+        **URL**: https://www.not-a-mozilla-browser.com/
```

Comment:
did you mean not gecko? ;)

```suggestion
        **URL**: https://not-gecko.example.com/
```

Reply:
good catch, :)